main.js

```diff
@@ -0,0 +1,34 @@
+const electron = require('electron')
```

Comment:
If we want to reuse (library) code both on server and client side, maybe we should agree to use semi-colons at end of lines?